spec/models/tag_spec.rb

```diff
@@ -30,6 +30,13 @@
       tag = build(:tag, name: Faker::Lorem.characters(number: 160))
       expect(tag).to be_valid
     end
+
+    context "when name is blank" do
+      it "is not valid" do
+        tag = build(:tag, name: "")
+        expect(tag).not_to be_valid
+      end
+    end
```

Comment:
Probably this test is already covered in the `ActAsTaggableOn::Tag` tests. I think we can remove it 🤔 .